ui/src/kapacitor/utils/alertMessageValidation.ts

```diff
@@ -1,24 +1,33 @@
 import _ from 'lodash'
 import {RULE_MESSAGE_TEMPLATE_TEXTS} from 'src/kapacitor/constants'
 
+// the following template variables must also pass validation, see #5492
+// they are not that much common to be part of RULE_MESSAGE_TEMPLATE_TEXTS
+const EXTRA_MESSAGE_VARIABLES = ['.Time.Unix', '.Time.UnixNano']
```

Comment:
should this be put into `src/kapacitor/constants` and imported?

Reply:
My preference is to keep the constants closer as much close to the code as possible, unless they are reused in multiple files ... which is not this case.